app/src/main/java/com/sdpteam/connectout/MapModel.java

```diff
@@ -0,0 +1,38 @@
+package com.sdpteam.connectout;
+
+import androidx.lifecycle.MutableLiveData;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MapModel implements InterfaceMapModel {
+    private ArrayList<Event> dataSet = new ArrayList<>(); //To be defined at a later stage
+
+    public MapModel() {
+        //initialize dataSet
```

Comment:
Empty constructor can be deleted

Reply:
I agree that it's useless at this stage, but my intention was to make sure that future implementations will initialize at creation. Still going to follow your tip though